src/util/moneystr.cpp

```diff
@@ -37,7 +37,7 @@ bool ParseMoney(const std::string& str, CAmount& nRet)
         return false;
     }
 
-    if (str.empty()) {
+    if (TrimString(str, " ").empty()) {
```

Comment:
Why not just check `!p[0]` below (after line 48)?

Reply:
TMTOWTDI :)  I find the current formulation more idiomatic, but I'll let others chime in and happily change if your opinion is the consensus opinion :)

Reply:
Why not the default pattern, which covers other whitespace?